rules/concepts.md

```diff
@@ -48,6 +48,154 @@ In some cases, it is useful to have a rule that is only manually triggered.
+| cron expressions | cron allows you to create nearly any schedule you can think of, e.g. every second sunday in November and December at 04:05 h. |
+| Time is Item     | It is a date and a time specified in an DateTime Item.                                                                        |
+| Time of Day      | It is a fixed time of the day, e.g. 09:00 h.                                                                                  |
+
+Time triggers do not provide any information in the [Available Values](#available-values).
+
+Please be aware that openHAB is using the [Quartz Scheduler](https://www.quartz-scheduler.org/documentation/quartz-2.2.2/), which is using a slighly different form than the Unix cron scheduler, for [cron expressions](https://www.quartz-scheduler.org/documentation/quartz-2.2.2/tutorials/tutorial-lesson-06.html).
+A Quartz cron expression takes the form of six or optionally seven fields:
+
+1. Seconds
+2. Minutes
+3. Hours
+4. Day of Month
+5. Month
+6. Day of Week
+7. Year (optional field)
+
+You may use the generator at [FreeFormatter.com](https://www.freeformatter.com/cron-expression-generator-quartz.html) or the openHAB WebUI rule setup to generate your cron expressions.
+
+### **Channel** Triggers
+
+Some add-ons provide trigger channels.
+
+Your rules can take actions based upon trigger events generated by these trigger channels.
+You can decide whether you want to catch only a specific or any trigger the channel provides.
+
+When a binding provides such channels, you can find the needed information in the corresponding binding documentation.
+There is no generic list of possible trigger events.
+The `triggerEvent`(s) available depend upon the specific implementation details of the binding.
+
+If the rule needs to know what the received event or the triggering channel was, use the [Available Values](#available-values).
+
+### **Thing** Triggers
+
+Your rules can take actions based upon status updates or status changes generated by Things.
+You can decide whether you want to catch only a specific or any status the Thing can get updated too.
+
+You can find all the possible values for status from [Thing Status](/docs/concepts/things.html).
+
+The `thingUID` is the identifier assigned to the Thing, manually in your configuration or automatically during auto discovery.
+You can find it from UI or from `.things` files.
+For example, one z-wave device can be "zwave:device:c5155aa4:node14".
+
+If the rule needs to know what the triggering Thing was, or access the previous or new status, use the [Available Values](#available-values).
+
+<!-- TODO: Update reference when actions are included in the reworked rule docs. -->
+Refer to [Thing Status Action](/docs/configuration/actions.html#thing-status-action) to find how to get the new thing status details or description in the script.
+
+### **System** Triggers
+
+There is only one System Trigger, the start level Trigger.
```

Comment:
This may not be true forever. There has been discussion about adding a rule loaded trigger and OH used to have a system shutdown trigger. To save our forgetting about it later I'd remove this sentence when/if more triggers are added I'd remove this sentence.

Reply:
To be ready for the future, I changed this to:

The **System** triggers include the system start level trigger.